app.js

```diff
@@ -0,0 +1,127 @@
+const statusDisplay = document.querySelector('.game--status');
```

Comment:
- [ ] It doesn't use global code. Let's encapsulate all DOM functionalities inside its own module, and expose it's api function for other modules to use. Creating a module for the DOM functionality is also very important for separating code logic from dom manipulation. See next point
 
- [ ] Keep the application logic separated from DOM manipulation tasks
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png)